04week/nasa/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE.html>
+  <html>
+    <head>
+      <link rel="stylesheet" type="text/css" href="css/style.css"/>
+    </head>
+
+
+    <body>
+      <header>
+        <div class="NASA"NASA> NASA </div>
+      </header>
+      <main>
+        <div class="box1">
+          <h2> Social Media</h2>
+            <divclass="boxx1"</div>
+          twitter-@nasa <br>
+          Facebook- fb.com /nasa <br>
+          instagram- @nasa<br>
+          youtube-youtube.com />NASA </div>
+
+          <div class="box2">
+              <h2> OFFICE INFO</h2>
+              <div class="boxx2"></div>
+              Public Communications Office<br>
```

Comment:
I would have used separate elements instead of BRs to allow for more control on styling.